runbotics/runbotics-scheduler/src/websocket/bot/process-launch/bot-process-instance.service.ts

```diff
@@ -136,7 +136,8 @@ export class BotProcessService {
         instanceToSave.trigger = processInstance.trigger;
         instanceToSave.triggerData = processInstance.triggerData;
         instanceToSave.parentProcessInstanceId = processInstance.parentProcessInstanceId;
-        return instanceToSave;
+
+        return { ...instanceToSave, user: processInstance.user };
```

Comment:
why do we need user here? I don't think it's wrong, so just asking for explanation to understand it better